pyne/alara.py

```diff
@@ -43,7 +44,7 @@
 
 def mesh_to_fluxin(flux_mesh, flux_tag, fluxin="fluxin.out",
                    reverse=False, sub_voxel=False, cell_fracs=None,
-                   cell_mats=None, print_progress=100000):
+                   cell_mats=None):
```

Comment:
can we still have a way for users to turn this off?

Reply:
@gonuke The commit https://github.com/pyne/pyne/pull/1401/commits/0dcce921c2ec13f4128c1d1beef4317970337d3f allow users to turn this off.